src/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./search-by-term');
```

Comment:
What's the purpose of `index.js` and `index.test.js` ? Tests will run for the required file even without it